tests/helpers/start-app.js

```diff
@@ -5,8 +5,8 @@ import config from '../../config/environment';
 export default function startApp(attrs) {
   let application;
 
-  let attributes = Ember.merge({}, config.APP);
-  attributes = Ember.merge(attributes, attrs); // use defaults, but you can override;
+  let attributes = Ember.assign({}, config.APP);
```

Comment:
Also, lets destructure above to avoid `Ember.assign` here.

Something like:

```js
const { assign } = Ember;

// ...snip...

let attributes = assign(/* stuff */);
```